unblob/processing.py

```diff
@@ -187,26 +198,27 @@ def _extract_chunk(
         outdir: Path,
         extractor: Extractor,
         result: TaskResult,
-    ):
-        # We only extract every blob once, it's a mistake to extract the same blog again
-        outdir.mkdir(parents=True, exist_ok=False)
+    ) -> bool:
+        # We only extract every blob once, it's a mistake to extract the same blob again
         try:
-            extractor.extract(inpath, outdir, result)
+            outdir.mkdir(parents=True, exist_ok=False)
+            extracted = extractor.extract(inpath, outdir, result)
+
+            fix_extracted_directory(outdir, result)
```

Comment:
We want to call `fix_extracted_directory` even in case of extraction errors.

Reply:
Finally it will not moved out to `_process_chunk`, but outside the `try` block - as it was previously